src/test/javascript/spec/component/programming-exercise/programming-exercise-update.component.spec.ts

```diff
@@ -403,6 +403,25 @@ describe('ProgrammingExercise Management Update Component', () => {
             route.url = of([{ path: 'import' } as UrlSegment]);
         });
 
+        it('should reset dates, id, project key and store zipFile', fakeAsync(() => {
```

Comment:
why does the test case say store zip file if it's an import from an existing exercise?

Reply:
Good catch, copy+paste error 😅 
Fixed!